lib/mrtrix3/_5ttgen/neonatal.py

```diff
@@ -22,11 +22,11 @@
 
 DHCP_CGM = list(range(5, 40))
 DHCP_SGM = [17, 18]
-DHCP_WM = [19] + range(40, 49) + range(51, 83) + range(85, 88)
+DHCP_WM = [19] + list(range(40, 49)) + list(range(51, 83)) + list(range(85, 88))
 DHCP_CSF = [49, 50, 83]
 DHCP_AMYG_HIPP = list(range(1, 5))
 
-MCRIB_CGM = list(range(1000, 1036) + range(2000, 2036))
+MCRIB_CGM = list(range(1000, 1004)) + list(range(1005,1036)) + list(range(2000, 2004)) + list(range(2005,2036))
```

Comment:
This relates to my question RE: absent / duplicated indices. Looks like the elements I thought may have been accidentally omitted you intended to omit; but I have some recollection that there were additionally indices that were duplicated in your original code. This is the better way to do it, but I was partly curious whether or not you would spot the influence of those duplications in a change in output of the script.

(P.S. Given it looks like an unusual omission, it might be worth adding a code comment indicating the structures omitted and why)

Reply:
Hi,

We have been working on this, these are the new changes:

- Now the algorithms are called mcrib and dhcp.
- The dHCP algorithm has been modified, now you just need to input the path of the derivates (/anat) folder and that's all, no mask required anymore. Note that there are some versions of the dHCP pipeline around with different names, I think now it covers all the possible cases.
- In the dHCP algorithm we added a filter to ensure that the merging of the sGM doesn't overlap with the CSF.
- A note has been added about the missing labels: the labels 1004 and 2004 don't exist in the original M-CRIB atlas.

As it is now, the dHCP algorithm is the same as in my original paper (with the addition of this CSF filter to ensure that the sGM doesn't enter in the CSF). Basically, it combines the tissue probability maps of the dHCP pipeline with the binary subcortical gray matter parcellations of the M-CRIB atlas. However, I've been thinking to modify it and add two possibilities: the use of the probability maps or not. This will translate in the addition of a new flag (-posteriors), that if specify it it will work as it does now, the only difference will be the use of tissue probability maps for the subcortical GM, in this way all the tissues will have the partial volumes. If not specify it, the algorithm will use only the binary segmentations for everything. I was thinking about this because probably a lot of the people that run the dHCP pipeline didn't use the -additional flag (for example in the dHCP data itself, this data is not there). What do you think? Does this make sense?